frontend/src/features/public-form/components/FormPaymentPage/stripe/components/PaymentBlock.tsx

```diff
@@ -181,7 +184,6 @@ export const StripePaymentBlock = ({
             S${centsToDollars(form?.payments_field?.amount_cents || 0)}
```

Comment:
```suggestion
            S${centsToDollars(form?.payments_field?.amount_cents ?? 0)}
```
I know this isn't part of this PR but I just saw it - shouldn't this use `??` instead? Because we only wanna default to 0 for nullish values
@foochifa since this was part of your commit some time ago haha

Reply:
I think for numbers (and if we trust that we will only accept numbers), it should be fine since the values that could be falsy (`undefined, null, 0`) should be treated as 0